server/src/completions/snippets.ts

```diff
@@ -0,0 +1,289 @@
+/**
+ * Inspired by bash-language-sever
+ * Repo: https://github.com/bash-lsp/bash-language-server
+ */
+
+import { InsertTextFormat, type CompletionItem, CompletionItemKind, MarkupKind } from 'vscode-languageserver'
+
+/* eslint-disable no-template-curly-in-string */
+export const SNIPPETS: CompletionItem[] = [
+  {
```

Comment:
Looks like something that could be generated procedurally from the doc. It might be simpler to maintain if we include the doc somewhere and add a script (maybe a [postinstall](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts)) that generates a file (maybe a json or a ts file) on installation

Reply:
Issue is this doc is not into the BitBake's folder, but into Poky: https://git.yoctoproject.org/poky/tree/documentation/ref-manual/tasks.rst

Reply:
Having a source file for the doc is indeed better for later maintenance. And seems like this file has more tasks than [here](https://docs.yoctoproject.org/singleindex.html#tasks) Section 6

Reply:
As answered by Ross, I will add the script that extracts the task documentation and the copy of the source doc file. The goal is now to keep the source doc and extract script up to date.

Reply:
> As answered by Ross, I will add the script that extracts the task documentation and the copy of the source doc file. The goal is now to keep the source doc and extract script up to date.

This change will be in another PR